card.go

```diff
@@ -179,3 +179,39 @@ func (c *Card) AddComment(text string) ([]byte, error) {
 	body, err := c.client.Post("/cards/"+c.Id+"/actions/comments", payload)
 	return body, err
 }
+
+// Archive will archive the card
+// https://developers.trello.com/advanced-reference/card#put-1-cards-card-id-or-shortlink-closed
+func (c *Card) Archive() ([]byte, error) {
```

Comment:
I'd like to add some real response types for all these methods (instead of `[]byte`). What's the response from Trello?